roles/openshift_hosted_templates/files/v3.7/origin/registry-console.yaml

```diff
@@ -27,7 +27,7 @@ objects:
         spec:
           containers:
             - name: registry-console
-              image: ${IMAGE_NAME}:${IMAGE_VERSION}
+              image: ${IMAGE_PREFIX}kubernetes:${IMAGE_VERSION}
```

Comment:
This is my primary issue.

We're going from something that is completely configurable to something that is hard-coding kubernetes into the image name.

If someone is publishing their own registry console image, it might be called something that doesn't include 'kubernetes' in the image name.

Reply:
I think the bit you may not be seeing here is [what this PR doesn't change about the enterprise template](https://github.com/sosiouxme/openshift-ansible/blob/37812f0da8f72df9108bfdcd882439d3faeea79b/roles/openshift_hosted_templates/files/v3.7/enterprise/registry-console.yaml#L30).

`              image: ${IMAGE_PREFIX}registry-console:${IMAGE_VERSION}`

Everything else that has to do this with a template (logging, metrics...) looks like this -- is already doing what you're saying you don't like. Only the origin registry-console templates are not yet . My idea was to change these templates to bring them in line with the de facto standard. Would you instead like to change the standard? Keeping in mind that this is pretty analogous to how oc and the server determine image names (`--template` or `oreg_url` supply `registry/namespace/prefix-${component}:${version}`, the component supplies a hardcoded `component` to fill in). The point is to have part of the image name come from shared context, and part to come from the specific thing that is being instantiated.

Reply:
I would argue, this template was right and the others are wrong.  Why make every character around the word kubernetes configurable and not kubernetes itself?

Reply:
So:

`image: ${IMAGE_PREFIX}${IMAGE_BASENAME}:${IMAGE_VERSION}`

would be OK?

Reply:
That would be fine with me.

Reply:
If this PR as is fixes the bug reported by QE then I'd like to go ahead and merge this. The mess of incongruency is sort of the hand we've been dealt with this component and we can work on fixing it in the future but since this PR is tied to a blocker I'd like to merge this unless this is actually making things worse.

Reply:
@sdodson it should fix the bug. Sorry for my delay; it's a small change to the PR to do what I just said but maybe it's preferable to do that as a followup?

Reply:
Whatever gets the bug fixed within the next 2 hours works for me.

Reply:
In trying to test my little followup change I'm wrestling with Ansible and losing.

So, my very biased vote... merge as-is.
Presumably needs backport PRs against 3.7 and 3.6?

Reply:
3.7 hasn't branched yet, but if we're fixing this for 3.6 yeah we need to backport to release-3.6. I'd wait until QE verifies the fix in 3.7 before bothering with that though.